package.json

```diff
@@ -86,7 +87,8 @@
     "@types/jasminewd2": "~2.0.8",
     "@types/node": "^12.19.9",
     "cordova-android": "^9.0.0",
-    "cordova-ios": "^6.1.1",
+    "cordova-ios": "6.1.1",
+    "cordova-plugin-advanced-http": "^3.1.0",
```

Comment:
Please add the cordova plugin reference to the `install.md` aswell as the npm reference